lib/Ed25519Signature2020.js

```diff
@@ -69,10 +71,10 @@ export class Ed25519Signature2020 extends LinkedDataSignature {
     }
 
     const contexts = [].concat(document['@context'] || []);
```

Comment:
This approach has been shown to be more performant:

```suggestion
    const context = document['@context'];
    const contexts = Array.isArray(context) ? context : [context];
```

Reply:
Thanks!